File: JS_Algo/프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    var answer = true;
    
    let pCount = 0
    let yCount = 0

    let str = s.toLowerCase()
    
    for (let i = 0; i < str.length; i++) {
        if (str[i] == 'p') {
            pCount += 1
        } else if (str[i] == 'y') {
            yCount += 1
        }
    }
    
    pCount == yCount ? answer=true : answer = false;
    
    return answer
}

File: SQL/프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int p = 0;
        int y = 0;
        
        for(int i = 0; i < s.length(); i++){
            
            if (s.charAt(i)=='p' || s.charAt(i)=='P'){
                p += 1;
                System.out.println(p + "p");
            }else if(s.charAt(i)=='y' || s.charAt(i)=='Y'){
                y += 1;
                System.out.println(y + "y");
                
            }
        }
        if(p == y){
            answer = true;
        }else{
            answer = false;
        }
        return answer;
    }
}